### DISCUSSION TITLE
Support for Sage 10 and Bedrock

### DISCUSSION THREAD

**Discussion by campaignupgrade** [welaika/wordmove#637] [Q&A] [answered]
Out of the box does wordmove support Sage 10 and Bedrock?

**Comment by alessandro-fazzi** [comment, marked as answer]
I know for sure that some users work with a bedrock setup.
I dont't know anything about Sage and somewhat think that moving a Sage theme or any other theme would (should?) be the same thing.

But what you mean with "out of the box" isn't of immediate understanding to me. E.g.: using bedrock you'll need to configure custom paths into `movefile.yml`; I don't know if you consider "out of the box" as "without any configuration" :)

**Comment by campaignupgrade** [comment]
If it can be configured with custom paths that's good enough! If it was hardwired with specific paths for the theme, plugins, etc, that would break it for Bedrock. Thanks!

**Reply by alessandro-fazzi** [reply]
You’re welcome.

If you mind, you could mark my reply as an answer, in order to keep things tidy and help others navigating the Q/As 😊
